src/tests/ftest/container/BasicTxTest.py

```diff
@@ -21,12 +21,13 @@
   Any reproduction of computer software, computer software documentation, or
   portions thereof marked with this legend must also reproduce the markings.
 '''
+from __future__ import print_function
 import os
 import time
 import traceback
 import sys
 import json
-from avocado import Test, main
+from avocado import Test
```

Comment:
(pylint-import-error) Unable to import 'avocado'

Reply:
Has this patch had master merged into to recently?  This should be fixed on recent master.